cv32e40s/tests/programs/custom/pmp/pmp.c

```diff
@@ -0,0 +1,16 @@
+// #include <stdio.h>
+// #include <stdlib.h>
+
+#include "pmp.h"
+
+int main(int argc, char *argv[])
+{
+  // reset_registers();
+  // default_full();
+  // default_none();
+  // mmode_only();
+  // napot_matching();
+  tor_macthing();
```

Comment:
We need a way to run all tests with "push-button" ease.
All of those tests should be runnable without having to edit the source files.